index.bs

```diff
@@ -1388,6 +1403,10 @@ optionally evidence of [=user consent=] to a specific transaction.
     ::  This member is intended for use by [=[RPS]=] that wish to select the appropriate authenticators to participate in
         the {{CredentialsContainer/create()}} or {{CredentialsContainer/get()}} operation.
 
+    :   <dfn>attestation</dfn>
+    ::  This member is intended for use by [=[RPS]=] that wish to express their preference among different attestation types. 
```

Comment:
these are not "attestation types", as  noted above in

Reply:
should use the term attestationPresentationPreference or attestationConveyancePreference here.